backend/wmg/pipeline/summary_cubes/cell_count.py

```diff
@@ -119,6 +128,49 @@ def create_cell_count_cube(corpus_path: str):
     logger.info(f"Cell count cube created and stored at {uri}")
 
 
+def add_missing_cell_types_to_df(df: pd.DataFrame):
```

Comment:
this function adds missing cell types (ancestors) to the dataframe stratified across non-numeric columns that are not cell type ontology term id